liquidcore/twofactor/models.py

```diff
@@ -22,6 +23,19 @@ class Invitation(models.Model):
     )
     code = models.CharField(max_length=200, default=random_code)
     expires = models.DateTimeField()
+    opened = models.BooleanField(default=False)
```

Comment:
this boolean field should be inferred from the `NULL` values of `opened_at` - we don't want duplicate fields on the model